cmd/skaffold/app/cmd/cmd.go

```diff
@@ -128,6 +128,7 @@ func setFlagsFromEnvVariables(commands []*cobra.Command) {
 
 func AddDevFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&opts.Cleanup, "cleanup", true, "Delete deployments after dev mode is interrupted")
+	cmd.Flags().StringArrayVarP(&opts.Watch, "watch-image", "w", nil, "Choose which artifacts to watch. Matches against image names. Default is to watch sources for all artifacts.")
```

Comment:
"Choose which artifacts to watch.  Artifacts with image names that contain the expression will be watched only. Default is to watch sources for all artifacts."